ts/src/pro/binance.ts

```diff
@@ -3792,7 +3792,8 @@ export default class binance extends binanceRest {
             client.reject (message, id);
         }
         // reset connection if 5xx error
-        if (this.safeString (code, 0) === '5') {
```

Comment:
this was an issue, as such approach didn't get first char of `code` instead it accepted `code` as array and tried to locate first key, which didnt work across all langs. the fixed line did it correctly